hudi-common/src/test/java/org/apache/hudi/common/table/read/TestHoodieFileGroupReaderBase.java

```diff
@@ -328,15 +328,24 @@ public void testSchemaEvolutionWhenBaseFilesWithDifferentSchema(HoodieFileFormat
     }
   }
 
+  private static Stream<Arguments> testArgsForDifferentBaseAndLogFormats() {
+    boolean supportsORC = supportedFileFormats.contains(HoodieFileFormat.ORC);
+    return Stream.of(
+        arguments(supportsORC ? HoodieFileFormat.ORC : HoodieFileFormat.PARQUET, "avro"),
+        arguments(HoodieFileFormat.PARQUET, "parquet")
```

Comment:
Should `arguments(HoodieFileFormat.PARQUET, "avro")` always be tested and `arguments(HoodieFileFormat.ORC, "avro")` be added only is ORC is supported?

Reply:
Adjusted the test